lms/static/sass/discussion/_discussion.scss

```diff
@@ -463,123 +271,6 @@ body.discussion {
     margin-bottom: $baseline;
   }
 
-  .responses {
```

Comment:
Context: These styles (466-582) were also removed from this deprecated partial and moved into _response.scss under a few different sections. I've noted what each section is here, ordered based on how they were defined in _discussion.scss. Their order is slightly adjusted now to aid readability in _responses.scss, and so I've added the table of contents section it applies to in parentheticals. 

First, the layout styling for responses (+wrapper - response list), the next bit is for the banners and labels applied to posts (+response - labels and banners), then the base styles for a single response element (+base - single response element), and finally the add response button (+element - add response area).